.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Test destream
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 5
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        sudo apt-get install p7zip rar zstd file libmagic1
+        python -m pip install --upgrade pip
+        pip install tox tox-gh-actions
+    - name: Test with tox
+      run: tox
```

Comment:
Missing EOF newline (sorry for that)

Reply:
Thanks. That's a good reason to add `pre-commit`.

Reply:
Yeah haha 😂  you referencing to that conversation on twitter?